### ISSUE TITLE
SUBSCRIBERS (5) count is incorrect or it doesn't allow me to see/unsub from dead accounts

### ISSUE THREAD

**Issue by xftroxgpx** [Minds/front#33]
In the following screenshot `SUBSCRIBERS (5)` should be `SUBSCRIBERS (1)`
OR, the other 4 users(or is the term `channels`?) should be shown so that I can potentially unsubscribe them from me, by ie. by blocking them first.

Note: if I remove the visible subscriber (by blocking then unblocking him and refreshing page) I do get `SUBSCRIBERS (4)` and an empty list of subscribers. I am guessing the 4 hidden/unshown subscribers are either banned or deactivated(themselves?).

![screenshot_2017-10-13_11-05-03](https://user-images.githubusercontent.com/26243389/31538697-ad05acf6-b006-11e7-9181-d10ec1968a81.png)

**EDIT:** added `SUBSCRIBERS (4)` screenshot:
![screenshot_2017-10-16_11-42-47](https://user-images.githubusercontent.com/26243389/31605693-28efdf00-b267-11e7-849e-eb0635be4ba9.png)

**Comment by xftroxgpx** [comment]
this is still a thing
which basically means that all the bots that were purged are still counted as subscribers/subscriptions, so I can understand the reluctance to fix this, as it looks like there are more people using the site than there actually are. (i think gab.ai suffers from the same issue :D or, at least it used to back when I last used it)

![screenshot_2018-10-18_02-10-27](https://user-images.githubusercontent.com/26243389/47123356-3b26a980-d27b-11e8-9ebb-775d73c0169d.png)

**Comment by xftroxgpx** [comment]
Closing because lost interest :D
If anyone still encounters this, create a new issue or (unlikely) ask Members to re-open this.
